src/main/content/start.html

```diff
@@ -339,10 +388,11 @@ <h3 class="eclipse_development_tools_title pull-left">Development builds</h3>
     <img id="bottomleftclouds" src="{{ "/img/bottom_left_clouds.png" | relative }}" alt="bottom left clouds">
     <img id="bottomrightcloud" src="{{ "/img/bottom_right_cloud.png" | relative }}" alt="bottom right cloud">
     <p id="cow_text">
-            Don't have a cow.
+        {% t start.download_package_section.cow_text %}
     </p>
     <p id="more_downloads_text">
-        More downloads are on the way.<br>
-        Check back soon!
+        {% t start.download_package_section.more_downloads_on_the_way %}
+        <br>
+        {% t start.download_package_section.check_back_soon %}
     </p>
-</div>
+</div>
```

Comment:
```suggestion
</div>

```

Add EOL to this file